libs/shared/components/src/Inputs/TokenInput.tsx

```diff
@@ -149,16 +150,25 @@ export const TokenInput = forwardRef<HTMLInputElement, TokenInputProps>(
                   </Typography>
                   {!hideMaxButton && (
                     <Button
-                      variant="text"
-                      color="inherit"
                       onClick={handleMaxClick}
                       disabled={maxDisabled}
                       sx={{
-                        minWidth: 0,
-                        padding: (theme) => theme.spacing(0.2, 1),
+                        display: 'flex',
+                        justifyContent: 'center',
+                        alignItems: 'center',
+                        borderRadius: 1,
+                        minWidth: 36,
+                        lineHeight: 1,
+                        color: 'text.secondary',
+                        padding: (theme) => theme.spacing(0.25, 0.5),
+                        background: (theme) =>
+                          alpha(theme.palette.common.white, 0.1),
+                        ':hover': {
+                          background: (theme) => theme.palette.grey[600],
+                        },
```

Comment:
Just for reference/example/info this would be replaced by the following approximation:
`flex items-center justify-center rounded w-12 leading-2 text-secondary py-1 px-2 bg-white/10 hover:bg-gray-600`

Reply:
yes, maybe just `min-w-9` instead of `w-12`,` leading-none` instead of `leading-2`, `py-0.5` and `px-1`